Author: Guillem Jover <gjover@sipwise.com>
Description: Use a proper accessor instead of messing with the struc directly
 which has become opaque in latest OpenSSL versions.

---
 apps/diameter_client/lib_dbase/tcp_comm.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/apps/diameter_client/lib_dbase/tcp_comm.c
+++ b/apps/diameter_client/lib_dbase/tcp_comm.c
@@ -74,7 +74,7 @@ long tcp_ssl_dbg_cb(BIO *bio, int oper,
   switch (oper) {
   case BIO_CB_WRITE: {
     char buf[256];
-    snprintf(buf, 256, "%s: %s", argp, bio->method->name);
+    snprintf(buf, 256, "%s: %s", argp, BIO_method_name(bio));
     INFO("%s", buf);
   } break;
 
